akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -32,7 +33,11 @@ class Member private[cluster] (
     case m: Member ⇒ uniqueAddress == m.uniqueAddress
     case _         ⇒ false
   }
-  override def toString = s"Member(address = ${address}, status = ${status})"
+  override def toString =
+    if (team == ClusterSettings.DefaultTeam)
+      s"Member(address = $address, status = $status)"
+    else
+      s"Member(address = $address, team = $team, status = $status)"
 
   def hasRole(role: String): Boolean = roles.contains(role)
 
```

Comment:
Managed to write a nice little test case with one gossip from each of two teams, where the team a node belonged to has removed it from the list of members and the other team still sees the node as in cluster. After merge the node should still be removed.

Struggling a bit to make the test pass, I think I need to pass the respective source teams of the gossip to the merge function and further to the `pickHighestPriority` but being a bit worried that I'm missing something obvious.